docs/mp/persistence.adoc

```diff
@@ -0,0 +1,1430 @@
+    ignored if the JPA provider you have <<JPA-Setup,set up>> is
+    Eclipselink.  See
+    https://docs.jboss.org/hibernate/orm/6.1/userguide/html_single/Hibernate_User_Guide.html#configurations-logging[Statement
+    logging and statistics] in the Hibernate ORM documentation for
+    more details about the `hibernate.show_sql` property.
+<3> This is an Eclipselink-specific property (and (a) is required and
+    (b) must be set to `false` if you are using Eclipselink), and will
+    be properly ignored if the JPA provider you have <<JPA-Setup, set
+    up>> is Hibernate ORM.  See
+    https://www.eclipse.org/eclipselink/documentation/3.0/jpa/extensions/persistenceproperties_ref.htm#weaving[weaving]
+    in the Eclipselink documentation for more details about the
+    `eclipselink.weaving` property.
+
+TIP: For an exhaustive list of Hibernate ORM-specific properties, see
+https://docs.jboss.org/hibernate/orm/6.1/userguide/html_single/Hibernate_User_Guide.html#configurations[Configurations]
+in the Hibernate ORM documentation.
+
+TIP: For an exhaustive list of Eclipselink-specific properties, see
+https://www.eclipse.org/eclipselink/documentation/3.0/jpa/extensions/persistenceproperties_ref.htm#sthref733[Persistence
+Property Extensions Reference] in the Eclipselink documentation.
+
+=== Usage [[JPA-Usage]]
+
+To use Helidon MP's Jakarta Persistence integration, once you have
+<<JPA-Setup, set up>> and <<JPA-Configuration, configured>> your
+project, you use the Jakarta Persistence APIs in almost the same
+manner as if your project were deployed to a Jakarta EE server.
+
+Specifically, you:
+
+1. Annotate your managed classes (entities, mapped superclasses, etc.)
+   appropriately (using
+   https://jakarta.ee/specifications/persistence/3.0/apidocs/jakarta.persistence/jakarta/persistence/entity[`@Entity`]
+   and similar annotations)
+
+2. Inject
+   https://jakarta.ee/specifications/persistence/3.0/apidocs/jakarta.persistence/jakarta/persistence/entitymanager[`EntityManager`]
+   instances appropriately with the
+   https://jakarta.ee/specifications/persistence/3.0/apidocs/jakarta.persistence/jakarta/persistence/persistencecontext[`@PersistenceContext`
+   annotation]
+
+3. Use an injected
+   https://jakarta.ee/specifications/persistence/3.0/apidocs/jakarta.persistence/jakarta/persistence/entitymanager[`EntityManager`]
+   to work with your managed objects
+
+In addition, you <<JTA-Usage, use Helidon MP's JTA integration>> to
+declare transactional boundaries where appropriate.
+
+A full tutorial of Jakarta Persistence is _well_ beyond the scope of
+this documentation.  Consult
+https://jakarta.ee/specifications/persistence/3.0/jakarta-persistence-spec-3.0.html#entities[the
+specification] for details on how to map your entity classes to
+relational database tables, and how to perform other related tasks.
+
+=== Examples [[JPA-Examples]]
+
+* https://github.com/oracle/helidon/tree/master/examples/integrations/cdi/pokemons[JPA
```

Comment:
This is hard-coding an example on master. Use the common attribute defined in includes/attributes.adoc